File: QE-Selenium-Java/Activity2.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String args[]){
        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/login-form/");
        System.out.println(driver.getTitle());
        driver.findElement(By.name("username")).sendKeys("admin");
        driver.findElement(By.name("password")).sendKeys("password");
        driver.findElement(By.cssSelector("body > div > main > div > div > div > div > div.mx-auto.my-5.flex.flex-col.justify-center.align-middle > form > button"));
        driver.quit();

    }
}

File: QE-Selenium-Python/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/login-form/")
    print(driver.title)
    driver.find_element(By.NAME,"username").send_keys("admin")
    driver.find_element(By.NAME,"password").send_keys("password")
    driver.find_element(By.CSS_SELECTOR,"body > div > main > div > div > div > div > div.mx-auto.my-5.flex.flex-col.justify-center.align-middle > form > button")
    driver.quit()